src/apis/getFollowData.ts

```diff
@@ -0,0 +1,20 @@
+import { Client } from '.';
+
+export interface User {
+  login: string;
+  id: number;
+  avatar_url: string;
+  url: string;
+}
+
+const PER_PAGE = 100;
+
+export const getFollowingList = async (token: string): Promise<User[]> => {
+  const { data } = await Client(token).get<User[]>(`/user/following?per_page=${PER_PAGE}`);
+  return data;
+};
```

Comment:
리턴 타입을 굳이 적어주지 않더라도, `axiosInstance`의 값을 리턴하는 `client.get`에 제네릭으로 타입을 주입해주었기 때문에 타입을 잘 추론해줄 것 같아요!

```suggestion
export const getFollowingList = async (token: string) => {
  const { data } = await Client(token).get<User[]>(`/user/following?per_page=${PER_PAGE}`);
  return data;
};
```